Makefile

```diff
@@ -0,0 +1,15 @@
+NAME    := freeipa
+SRC_EXT := gz
+SOURCE  = https://releases.pagure.org/$(NAME)/$(NAME)-$(VERSION).tar.$(SRC_EXT)
+DL_VERSION = 4.6.3
+DSOURCE = https://releases.pagure.org/$(NAME)/$(NAME)-$(DL_VERSION).tar.$(SRC_EXT)
+PATCHES = 0001-release-4-6-3-4-gfe5d037.patch
```

Comment:
I guess this also relates to the comment on the patch file itself.

Reply:
Renamed the patch file from the name that git-describe came up with for it.

Reply:
I'm not sure that is much more clear than before, given that `git describe` is just using whatever tag format the git repo is using and people (not familiar with this repos particular git tagging format) looking at patch file name are probably going to be thinking more in terms of public release numbers, with dots in them

I guess we don't need to belabour this.